src/main/java/com/example/codeE/ulti/Constant.java

```diff
@@ -0,0 +1,6 @@
+package com.example.codeE.ulti;
+
+//all constant string will be declared here
+public class Constant {
+    public static final String FILE_PATH = "src/main/resources/excel/";
```

Comment:
This should not be here. What's the point of setting a file path, which is also pointing into the project, here?